tox.ini

```diff
@@ -12,7 +18,8 @@ python =
 [testenv]
 commands = python run_tests.py
 deps =
-    flake8 >= 5.0.0
+    flake85: flake8 >= 5.0.0, < 6.0.0
+    flake86: flake8 >= 6.0.0, < 7.0.0
```

Comment:
Again the caps here don't seem productive since we then can't start easily testing against 7 or later.

Reply:
The `< 7.0.0` might not be productive, indeed, but it is abundantly clear on what is that "requirement group" is.

Given that the `flake86` is an alias for `flake8 v6`, I thought it'd be counter-intuitive to write it as `flake86: flake8 >= 6.0.0`, since it will pick up v7 "at its earliest convenience" (which might also fail, so the report will be weird).
Writing `flake85` as `flake8 >= 5.0.0` is definitely wrong, since it will pick-up `flake8 v6`, and **definitely** not verify compatibility with v5 (which some extensions require at the moment).

I will fix the cap at _`setup.py`_, as you requested; however, "I'd like to urge you to reconsider" this decision in testing.

Reply:
I'm not sure about the way to model this in tox, but we could perhaps also have a `flake8` or `flake8-latest` dependency alias that always uses the latest (uncapped) flake8 version. That would give us a complete test matrix for today and tomorrow.

Reply:
> we then can't start easily testing against 7 or later.

I am not sure were is this coming from. Testing v7 will be as easy as:
```
    flake87: flake8 >= 7.0.0, < 8.0.0
```

-----

When v7 comes out. It will be a manual action, for sure.
or, idk - I can add a `flake8next` target, which is 100% uncapped?

In that case, for all `py` targets, `tox` will check (at CI-time ofc) that the latest `pip install flake8` for each python version works.

(PS: I actually like that idea now. Please let me know of your thoughts)

Reply:
>  or, idk - I can add a flake8next target, which is 100% uncapped?

This is what I poorly advocated for while on my phone earlier